packages/core/src/pending_tasks.ts

```diff
@@ -53,6 +53,12 @@ export class PendingTasksInternal implements OnDestroy {
     if (this._hasPendingTasks) {
       this.hasPendingTasks.next(false);
     }
+    // We call `unsubscribe()` to release observers, as users may forget to
+    // unsubscribe manually when subscribing to `isStable`. We do not call
+    // `complete()` because it is unsafe; if someone subscribes using the `first`
+    // operator and the observable completes before emitting a value,
+    // RxJS will throw an error.
+    this.hasPendingTasks.unsubscribe();
```

Comment:
```suggestion
    if(!this.hasPendingTasks.closed) {
      this.hasPendingTasks.unsubscribe();
    }
```

Should probably be guarded in case it's already closed.